src/utils/diagnostic.ts

```diff
@@ -0,0 +1,70 @@
+import { Diagnostic, DiagnosticSeverity, Range } from 'vscode-languageserver';
+
+const ParseErrorExp = /^Parse error on line (\d+)/;
+const OnLineErrorExp = / \(on line (\d+)\)\.$/;
+
+export interface TemplateLinterError {
+  moduleId: string;
+  message: string;
+  source: string;
+  line?: number;
+  column?: number;
+}
+
+export function toDiagnostic(source: string, error: TemplateLinterError): Diagnostic {
+  return {
+    severity: DiagnosticSeverity.Error,
+    range: toRange(source, error),
+    message: toMessage(error),
+    source: 'ember-template-lint'
```

Comment:
`atom-languageclient` ignores the source and shows the a different name. But I did not add the name `ember-template-lint`. It's been like this before.